### ISSUE TITLE
Possible Improvements to server/client

### ISSUE THREAD

**Issue by FerLuisxd** [ricochhet/Erupe#58]
A guy created a pastebin with stuff we can add to the docs.
Also with updates to some of the server/client files: https://pastebin.com/QqAwZSTC
There is also a youtube guide of how to do all the steps. Maybe you guys can make use of some of this info as I don't know if they have a repo

**Comment by Stefan240** [comment]
There seems to be some useful Info it seems. 

There are a few problems with just adding it here. 

The Text must be vetted for stuff we cannot include here for legal reasons (like quest-files exes and stuff) 

Also before we could any of the stuff written there permission of the author of that pastebin post would be needed. 

Also one would need to check if all of what is listed there also applies to the version of erupe that is hosted in this repo. It seems to me that the person (or maybe people) work on their own on the server. 

Kinda sad that there can not be an open and unified attempt at creating a server. But since I am not really coding stuff I will refrain from that discussion. 

Anyway thank you for sharing the resource. If you find something that might of interest for this repo as well, feel free to comment the line number. 

But the problems with permissions and stuff would be needed to be sorted before adding anything here.

**Comment by FerLuisxd** [comment]
Yeah my point here is to unify the progress, I know there can be some possible solution to this.
Recently talked to the author, they have a [repo](https://github.com/xl3lackout/Erupe) (which is not a fork but a reupload?) 
Maybe you guys can reach to them so they can unify progress rhobiusvoid#2191 or ask for permission

**Comment by ricochhet** [comment]
That repo largely seems like their own fork. I'll consider messaging them on Discord. I'm pretty sure there's a few forks out there though. So I'm a bit against merging any one or the other. This one is largely still the one that was deleted, with very slight modifications for ease of use.
